package androidx.compose.material.icons.filled;

import g1.m0;
import g1.t;
import j0.a;
import java.util.ArrayList;
import k0.c;
import k1.a0;
import k1.e;
import k1.f;
import k1.h0;
import k1.j;
import k1.m;
import k1.n;
import k1.s;
import kotlin.jvm.internal.l;

/* JADX INFO: compiled from: r8-map-id-cf66f36031e302b9a7b5a76da57d208ae1b6ba4a38bba3bee1d4ed6983c54df3 */
/* JADX INFO: loaded from: classes.dex */
public final class HideImageKt {
    private static f _hideImage;

    public static final f getHideImage(a aVar) {
        l.f("<this>", aVar);
        f fVar = _hideImage;
        if (fVar != null) {
            return fVar;
        }
        e eVar = new e("Filled.HideImage", 24.0f, 24.0f, 24.0f, 24.0f, 0L, 0, false, 224);
        int i10 = h0.f9067a;
        long j = t.f6905b;
        m0 m0Var = new m0(j);
        ArrayList arrayList = new ArrayList(32);
        arrayList.add(new n(21.0f, 5.0f));
        arrayList.add(new s(0.0f, -1.1f, -0.9f, -2.0f, -2.0f, -2.0f));
        arrayList.add(new k1.l(5.83f));
        arrayList.add(new m(21.0f, 18.17f));
        arrayList.add(new a0(5.0f));
        arrayList.add(j.f9075c);
        e.a(eVar, arrayList, 0, m0Var);
        m0 m0Var2 = new m0(j);
        bj.n nVarC = k0.a.c(2.81f, 2.81f, 1.39f, 4.22f);
        nVarC.l(3.0f, 5.83f);
        nVarC.s(19.0f);
        nVarC.i(0.0f, 1.1f, 0.9f, 2.0f, 2.0f, 2.0f);
        nVarC.k(13.17f);
        nVarC.m(1.61f, 1.61f);
        gb.e.p(nVarC, 1.41f, -1.41f, 2.81f, 2.81f);
        nVarC.n(6.0f, 17.0f);
        nVarC.m(3.0f, -4.0f);
        nVarC.m(2.25f, 3.0f);
        nVarC.m(0.82f, -1.1f);
        c.u(nVarC, 2.1f, 2.1f, 6.0f);
        e.a(eVar, nVarC.f3166a, 0, m0Var2);
        f fVarB = eVar.b();
        _hideImage = fVarB;
        return fVarB;
    }
}
